pages/reference-architecture/_sub-hero.html

```diff
@@ -2,39 +2,29 @@
-      best-practices way to assemble the code from the
-      <a href="/infrastructure-as-code-library/">Infrastructure as Code Library</a>
-      into an end-to-end tech stack that includes just about everything you
-      need: server cluster, load balancer, database, cache, network topology,
-      monitoring, alerting, CI/CD, secrets management, VPN, and more (check out
-      the <a href="/devops-checklist/">Production Readiness Checklist</a> to see
-      what it takes to go to prod).
-    </p>
-    <p>
-      We generate the Reference Architecture based on your needs, deploy into your
-      AWS accounts, and give you 100% of the code. Since you have all the code, you
-      can extend, enhance, and customize the environment exactly according to your
-      needs. The deploy process takes about a day. <a href="/contact">Contact Us</a>
-      to set up a demo!
-    </p>
-    <p>
-      We also offer a
-      <a href="https://www.cisecurity.org/benchmark/amazon_web_services/" target="_blank">
-        CIS AWS Foundations Benchmark
-      </a> compliant version of the Reference Architecture. See our
-      <a href="/achieve-compliance">Compliance offering</a> to learn more.
+      The Reference Architecture is an opinionated, battle-tested, best-practices way of setting up all the foundational pieces you need to get started with AWS and Terraform, including:
+      <ul>
+        <li>AWS multi-account structure: <a href="/landing-zone-for-aws/">Gruntwork Landing Zone</a>.</li>
+        <li>Network topology: VPCs, subnets, route tables, VPN, etc.</li>
+        <li>CI / CD: <a href="/pipelines">Pipelines</a>.</li>
+        <li>Auth: AWS Identity Center (SSO), IAM roles, OIDC, etc.</li>
+        <li>Guard Rails: SCPs, AWS Config, GuardDuty, CloudTrail, etc.</li>
+        <li>Compliance: <a href="/achieve-compliance/">Gruntwork Compliance</a>.</li>
+      </ul>
     </p>
+      <p>
+        We generate the Reference Architecture based on your needs, deploy into your AWS accounts, and give you 100% of the code. Since you have all the code, you can extend, enhance, and customize it as much as you need. The deploy process takes about a day. <a href="/contact">Contact Us</a> to set up a demo!
+      </p>
     <p
       class="alert"
       style="border: 1px solid #194c5f; background-color: #242e3b;"
     >
-      <strong>Get a Detailed Walkthrough of the Reference Architecture Account Setup</strong
+      <strong>Get a Detailed Walkthrough of the Reference Architecture Account Structure</strong
       ><br />
       See our guide
       <a
-        href="https://docs.gruntwork.io/refarch/configuration/"
-        >Get Started with the Gruntwork Reference Architecture</a
+        href="https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/"
+        >How to configure a production-grade AWS account structure using Gruntwork AWS Landing Zone</a
       >.
     </p>
```

Comment:
Please remove this callout entirely.

Reply:
Do you mean to remove this whole section? 
<img width="951" alt="Zrzut ekranu 2023-07-4 o 12 32 53" src="https://github.com/gruntwork-io/gruntwork-io.github.io/assets/72336345/dbbe355d-0466-4779-8815-4db11927c7bc">

Reply:
Huh, I thought I saw a comment asking which callout, but now I don't see it... Well, just in case, I meant we should remove this call out, as the article it links to has been changed/refactored/removed:

![Screen Shot 2023-07-05 at 11 17 02 AM](https://github.com/gruntwork-io/gruntwork-io.github.io/assets/711908/b78e4703-dc67-45da-a552-6765d6582a24)